File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseTimeMapBoundingBoxesResult.scala

package org.openapitools.client.model


case class ResponseTimeMapBoundingBoxesResult (
    _searchId: String,
    _boundingBoxes: List[ResponseBoundingBox],
    _properties: ResponseTimeMapProperties
)
object ResponseTimeMapBoundingBoxesResult {
    def toStringBody(var_searchId: Object, var_boundingBoxes: Object, var_properties: Object) =
        s"""
        | {
        | "searchId":$var_searchId,"boundingBoxes":$var_boundingBoxes,"properties":$var_properties
        | }
        """.stripMargin
}

File: fsharp-giraffe-server/traveltimeplatform/src/model/ResponseTimeMapBoundingBoxesResult.fs
namespace traveltimeplatform.Model

open System
open System.Collections.Generic
open traveltimeplatform.Model.ResponseBoundingBox
open traveltimeplatform.Model.ResponseTimeMapProperties

module ResponseTimeMapBoundingBoxesResult = 

  //#region ResponseTimeMapBoundingBoxesResult


  type ResponseTimeMapBoundingBoxesResult = {
    SearchId : string;
    BoundingBoxes : ResponseBoundingBox[];
    Properties : ResponseTimeMapProperties;
  }
  //#endregion
  